cibuildwheel/__main__.py

```diff
@@ -308,7 +328,7 @@ def print_preamble(platform: str, build_options: BuildOptions) -> None:
 
 
 def print_build_identifiers(
-    platform: str, build_selector: BuildSelector, architectures: Set[Architecture]
+        platform: str, build_selector: BuildSelector, architectures: Set[Architecture]
```

Comment:
Oops, vi did this, not me! I would never un-Black something! 😳

(Okay, I have been trying to convert `"` -> `'` when I see them to be consistent; most cases of `"` are my fault and because Black has finally retrained my brain to use `"`, I used to be a single-quoter.)